composer.json

```diff
@@ -18,7 +18,7 @@
         "source": "https://github.com/cakephp/cakephp"
     },
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.6.0",
```

Comment:
@admad the dropbox link is mine for the drivers in appveyor. may I update you PR or we first merge this one and I'll do an other one